### DISCUSSION TITLE
Mention Ruby wrapper?

### DISCUSSION THREAD

**Discussion by gjtorikian** [pemistahl/lingua-rs#299] [Show and tell]
Hiya! I noticed you had a Python wrapper listed in your README. I just wanted to offer that I have done a Ruby wrapper myself, and am currently running it in production: https://github.com/gjtorikian/what_you_say

Just in case you wanted to call it out. Thanks for the project!

**Comment by pemistahl** [comment]
Hi Garen, thank you for letting me know about your Ruby wrapper. That's great. I will add a note to the README.
